css/Comments.css

```diff
@@ -0,0 +1,3 @@
+.bulkmanagerheading .title {
```

Comment:
This will hide it everywhere... is this necessary? Other gridfields will use bulk manager.